src/api/routes/cron.ts

```diff
@@ -39,6 +42,28 @@ const cron = router({
     });
     await Promise.all(promises);
   }),
+
+  /**
+   * Check the numbers of meetings in progess by each Tencent user id
+   */
+
+  currentMeetingNumbers: procedure.mutation(async () => {
+    const TMuserID: string = "ADMIN"; // change if multiple users used, possibly using hashed TM user Ids
+    const currentMeetingList = await getCurrentMeetings();
```

Comment:
spare the intermediate variable. `const count = (await listMeetings()).meeting_info_list...`.